app/_gateway_entities/vault.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Vaults
+content_type: reference
+entities:
+  - vault
+
+description: A Vault is used to store secrets.
+
+related_resources:
+  - text: Vault CLI
+    url: /
+  - text: Secrets Management
+    url: /secrets-management
+  - text: Secret rotation
+    url: /gateway/secret-management/secret-rotation
+  
+
+faqs:
+  - q: I'm using declarative configuration for my Vault. Should I split my Vault configuration?
+    a: |
+        For larger teams with many contributors, or organizations with multiple teams, we recommend splitting Vault configuration and managing it separately. We recommend splitting the configuration for the following reasons:
+        * Vault are closer to infrastructure than other {{site.base_gateway}} configurations.
+        Separation of routing policies from infrastructure-specific configurations helps
+        keep configuration organized.
+        * Vaults may be shared across teams. In this case, one specific team shouldn't
+        control the Vault's configuration. One team changing the Vault a can have
+        disastrous impact on another team.
+        * If a Vault is deleted while in use -- that is, if there are still references to
+        secrets in a Vault in configuration -- it can lead to total loss of proxy capabilities.
+        Those secrets would be unrecoverable.
+  - q: How should I manage my Vault configuration with decK?
+    a: |
+        To keep your environment secure and avoid taking down your proxies by accident, make sure to:
+        * Manage Vaults with distributed configuration via tags.
+        * Use a separate [RBAC role, user, and token](/gateway/entities/rbac/)
+        to manage Vaults. Don't use a generic admin user.
+        * Set up a separate CI pipeline for Vaults.
```

Comment:
I found the source of this: 
![image](https://github.com/user-attachments/assets/f6a8ffe8-dd81-4ae7-af0e-ef1c124a16be)

I don't think this actually answers the Q here. This sounds more like "what are some best practices for managing Vaults"